meteor-app/imports/ui/service/summary-tab.js

```diff
@@ -0,0 +1,164 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import moment from 'moment'
+import Alert from '/imports/ui/utils/alert'
+import { Button, List, Accordion, Icon, Grid, Loader } from 'semantic-ui-react'
+
+import '/imports/ui/layouts/assessment.css'
+import {
+  LOG_EVENT_READABLE,
+  STATUS_UPDATE,
+  MECHANIC_UPDATE,
+  NEW_JOB,
+  PHONE_CALL,
+  SEND_SMS,
+  JOB_STATUS,
+  JOB_STATUS_READABLE,
+  JOB_STATUS_BUTTON,
+  JOB_STATUS_STYLES,
+  LOG_EVENT_TYPES
+} from '/imports/api/constants'
+import printJobCard from '/imports/ui/assessment/assessment-print-job'
+import MechanicModal from '../assessment/mechanic-modal'
+import SmsModal from '../assessment/sms-modal'
+import PhoneModal from '../assessment/phone-modal'
+
+const Summary = props => {
+  const { status, jobNo, bikeDetails, services, mechanic, pickupDate, totalCost, customerDetails } = props.job
+
+  conole
+  // const servicePackage = services.baseService
+  // const pickupDisplay = moment(pickupDate).format('D/M/YYYY')
```

Comment:
I am not a fan of commenting out code. Git is excellent at tracking changes, so be confident, and don't leave outdated code lying around :)